.gitignore

```diff
@@ -57,6 +57,7 @@ venv/*
 src/covid19.py
 src/covid19_wrap.c
 
+*.ipynb
```

Comment:
There are some notebooks included in the examples
This will ignore any new ones being added

Reply:
Removed this ignore.